Detectors/TOF/reconstruction/src/Encoder.cxx

```diff
@@ -256,7 +259,7 @@ bool Encoder::encode(std::vector<std::vector<o2::tof::Digit>> digitWindow, int t
   auto start = std::chrono::high_resolution_clock::now();
 
   mEventCounter = tofwindow; // tof window index
-  mIR.orbit = mEventCounter / Geo::NWINDOW_IN_ORBIT;
+  mIR.orbit = mEventCounter / Geo::NWINDOW_IN_ORBIT + o2::raw::HBFUtils::Instance().orbitFirstSampled;
```

Comment:
I am still not sure that this is correct: orbitFirstSampled is the orbit from which the sampling starts, the actual 1st orbit seen in the collision-context might be >  orbitFirstSampled. I think one should use `HBFUtils::Instance().getFirstSampledTFIR().orbit` instead, since this will give the 1st orbit of the TF where the digitization starts.
Could you run e.g. with  `HBFUtils.orbitFirst=13700;HBFUtils.orbitFirstSampled=13800` 
to avoid the situation were the `orbitFirstSampled` and `HBFUtils::Instance().getFirstSampledTFIR().orbit`  coincide?

Reply:
Yes, I will test it (thanks also for the explanation, I didn't get the difference between the two options)

Reply:
orbitFirstSampled -> firsts orbit in digits 13800
getFirstSampledTFIR -> first orbit in digits 13700

So I guess the second one is correct.

In any case in our digitization we are using orbitFirstSampled to start to fill the timeframe (I didn't test the original digitization here), Shell I change also it there?


I mean here:
void Digitizer::init()
{

  // set first readout window in MC production getting
  mReadoutWindowCurrent = o2::raw::HBFUtils::Instance().orbitFirstSampled * Geo::NWINDOW_IN_ORBIT;

Reply:
Not sure since the digitization does not know about TFs.
What is the exact meaning of the mReadoutWindowCurrent? 
The orbitFirstSampled can be anywhere within the TF, it is just an internal hint to the IR sample from which orbit to start (and the real 1st interaction orbit will be likely >  orbitFirstSampled). 
As such, this is the smallest orbit you *may* encounter in the collision context.

Reply:
Then it should be ok, since you are guaranteed not to have interactions before the  orbitFirstSampled

Reply:
Good. PR is ready now